### PR TITLE
Improve integration tests

### PR DESCRIPTION
**Pull request by gbudau**
Create conftest.py to reuse fixtures
Update tests
Update default.conf

### CODE REVIEW

**Review by Damrod — APPROVED** [review]
Very much cleaner now, thank you!

File: srcs/app/test/integration/conftest.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    if os.path.exists(filepath):
+        os.remove(filepath)
+
+@pytest.fixture(scope='function')
+def random_dir(random_filename):
+    dirpath = TMP_TEST_DIR + random_filename
+    os.makedirs(dirpath, exist_ok=True)
+    yield dirpath
+    if os.path.exists(dirpath):
+        os.rmdir(dirpath)
```
Comment (vdedios) [inline srcs/app/test/integration/conftest.py:53]: I like this! Also though to use `os` capabilities to create/delete dirs :)

**Review by vdedios — APPROVED** [review]
Great job! I also like the ide of having fixtures in a separate file. Altough if tests started growing in size you'll end up mixing up unrelated fixtures. In that case I'd separate them but, as said, in our situation this is the cleanest. Thank you :smile:

File: srcs/test/integration/conftest.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import os
+from pathlib import Path
+import pytest
+import shutil
+import subprocess
+import tempfile
+import time
+import uuid
+
+PROJ_DIR = str(Path(__file__).parents[4])
```
Comment (Damrod) [inline srcs/test/integration/conftest.py:10]: parents[3] skills fix the tests

Comment (gbudau) [reply srcs/test/integration/conftest.py]: :+1: Thank you!

**Comment by gbudau** [comment]
Thanks you! :tada:

**Pull request merged**